docs/pages/core-concepts.mdx

```diff
@@ -0,0 +1,224 @@
+[authentication](./architecture/authentication.mdx) work in Teleport.
+
+### Teleport Proxy Service
+
+The **Teleport Proxy Service** allows for secure access to resources in a your
+infrastructure from the public internet without the need for a VPN. 
+
+It establishes reverse tunnels to the **Teleport Auth Service** and **Teleport
+Services**, which can run in private networks. This means that, in the Proxy
+Service's minimal configuration, you can expose only port `443` to the internet
+and run the rest of your infrastructure in private networks.
+
+You can also configure clients to bypass Proxy Service instances and connect to
+resources with Teleport-issued certificates directly. 
+
+Read our guide to [how the Teleport Proxy Service
+works](./architecture/proxy.mdx).
+
+## Teleport Services
+
+A **Teleport Service** manages access to resources in your infrastructure, such
+as Kubernetes clusters, Windows desktops, internal web applications, and
+databases.
+
+A single running `teleport` binary can run one or more **Teleport Services**,
+depending on the user's configuration. Read about all subcommands of `teleport`
+in our [CLI Reference](./reference/cli.mdx#teleport).
+
+### Teleport Application Service
+
+Proxies HTTP and TCP traffic to user-configured endpoints, e.g., internal web
+applications or the AWS Console.
+
+Read more about the [Teleport Application
+Service](./application-access/introduction.mdx).
+
+### Teleport Database Service
+
+Proxies TCP traffic in the native protocols of popular databases, including
+PostgreSQL and MySQL. 
+
+Read more about the [Teleport Database
+Service](./database-access/introduction.mdx).
+
+### Teleport Desktop Service
+
+Proxies Remote Desktop Protocol traffic to Windows desktops.
+
+Read more about the [Teleport Desktop
+Service](./desktop-access/introduction.mdx).
+
+### Teleport Kubernetes Service
+
+Proxies HTTP traffic to the Kubernetes API server.
+
+Read more about the [Teleport Kubernetes
+Service](./kubernetes-access/introduction.mdx)
+
+### Teleport SSH Service 
+
+An SSH server implementation that allows users to execute commands on remote
+machines while taking advantage of Teleport's built-in access controls,
+auditing, and session recording.
+
+Read more about the [Teleport SSH Service](./server-access/introduction.mdx).
+
+### Machine ID
+
+Allows machines and services—called **bot users**—to communicate securely with
+resources in your infrastructure by automatically provisioning and renewing
+credentials. 
+
+Bot users can connect to resources in your infrastructure without relying
+on static credentials (e.g., certificates and private keys) that become more
+vulnerable to attacks the longer they remain in use.
+
+Unlike other **Teleport Services**, Machine ID runs via the `tbot` binary,
+rather than the `teleport` binary.
+
+Read more in our [Machine ID guide](./machine-id/introduction.mdx).
+
+### Agent
```

Comment:
With the push towards agentless are we sure we want to formally define agent here?

Reply:
I was looking for a term we could use to describe an instance of a Teleport service. We already use "agent" in the docs, the Teleport source, and conversations with users/prospects, so this seemed appropriate. I know there are misconceptions around the use of agents in Teleport (https://github.com/gravitational/teleport/issues/13351), but I thought we could add clarity to the term here rather than choose a different term.

I also think the term "agent" is better than alternatives. While we could reappropriate "Node" to mean "any service that manages access to a resource", I think the "Node/node" distinction is unwieldy. We could say "Service instance", but it has twice the words/syllables. There's also "relay", as per StrongDM, but that doesn't really describe the SSH Service.

What do you think?

Reply:
I don't feel strongly about this, totally up to you. Agent is fine with me.